network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+* **TCP protocol**: The Load Balancer will attempt to initiate a TCP (layer 4) connection to the backend server. 
+
+### Choosing a Verify Certificate setting
+
+If you select the HTTPS protocol, you are prompted to choose a Verify Certificate setting. Note that this setting does not display if you selected HTTP or TCP protocol.
+
+When the Load Balancer initiates an HTTPS connection to a backend server, the backend server presents its certificate. 
+- When Verify Certificate is selected, the Load Balancer checks the backend server's certificate and only connects if it is valid.
+- When Verify Certificate is unselected, the Load Balancer does not check the backend server's certificate, connecting to the backend server regardless of the certificate's validity. 
+
+This setting mostly concerns those using self-signed certificates on the backend server/s.
+
+Note that the same Verify Certificate setting you select here will also be used for health checks, if you choose HTTPS for your [health check method](#choosing-a-health-check-method).
+
+### Choosing a proxy protocol
+
+Proxy protocol determines how connection information from the client (eg the client’s IP address) will be transferred through the Load Balancer and on to the destination backend server. This allows the backend server to know the client's address, whatever the upper layer protocol (HTTP/HTTPS/TCP). 
+
+* **No proxy protocol**:  Disable proxy protocol.
+* **Proxy protocol v1**: Version one (text format).
+* **Proxy protocol v2**: Version two (binary format).
+* **Proxy protocol v2-ssl**: Version two with SSL connection
+* **Proxy protocol v2-ssl-cn**:  Version two with SSL connection and common name information.
+
+### Choosing a health check method
+
+Load Balancers should only forward traffic to “healthy” backend servers. To monitor the health of a backend server, health checks regularly attempt to connect to backend servers using the defined protocol to ensure that servers are up and listening. Various protocols for health checks are available:
+
+* **HTTP**: When selecting HTTP heath checks, you are prompted to choose a **method** (`GET`, `POST` or `PUT`), the **URI** to be targeted, and the response **code** that signals a successful health check.
+* **HTTPS**: The same method and fields as for an HTTP health check apply, but data transmission is encrypted. If you chose HTTPS as the main protocol for the Load Balancer's backend, the Verify Certificate setting you chose there is inherited for health checks. If you chose a different protocol for the Load Balancer's backend, the backend server's certificate will **not** be verified during health checks. See the [dedicated Verify Certificate section](#choosing-a-verify-certificate-setting) for more information.
```

Comment:
As we recently clarified, the verify certificate option won't be displayed in the HTTPS health check section. Now the behavior is as follows:

- If the backend protocol is HTTPS, then the user can set the verify certificate option and the same is applied to HTTPS health checks but nothing particular is displayed
- If the backend protocol is something different from HTTPS, then in this case the HTTPS health check automatically disables certificate verification --> it trusts any certificate, even self signed ones
A note maybe, if the certificate validation fails, then the health check fails and the backend server will be considered unhealthy.

Reply:
To me, what is already written is therefore appropriate? It already says the things in your comment, I think, apart from the note.